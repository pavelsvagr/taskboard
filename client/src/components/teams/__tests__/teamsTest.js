/* eslint react/jsx-filename-extension:0 */
import React from "react"
import ReactDOM from "react-dom"
import fetchMock from "fetch-mock"
import thunk from "redux-thunk"
import configureMockStore from "redux-mock-store"
import { Provider } from "react-redux"
import { MemoryRouter } from "react-router-dom"
import renderer from "react-test-renderer"
import Teams from "../Teams"
import testStore from "../../../helpers/tests/testAdminStore"
import testUserStore from "../../../helpers/tests/testUserStore"
import windowMedia from "../../../helpers/tests/windowMedia"

const middlewares = [thunk]
const mockStore = configureMockStore(middlewares)

describe("Teams.jsx", () => {
  afterEach(() => {
    fetchMock.restore()
  })

  it("SMOKE: render", () => {
    const store = mockStore(testStore)
    Object.defineProperty(window, "matchMedia", windowMedia)

    const div = document.createElement("div")
    ReactDOM.render(
      <Provider store={store}>
        <MemoryRouter>
          <Teams />
        </MemoryRouter>
      </Provider>, div)
  })

  it("Snapshot: Match admin", () => {
    const store = mockStore(testStore)
    Object.defineProperty(window, "matchMedia", windowMedia)

    const tree = renderer
      .create(
        <Provider store={store}>
          <MemoryRouter>
            <Teams />
          </MemoryRouter>
        </Provider>
      )
      .toJSON()

    expect(tree).toMatchSnapshot()
  })

  it("Snapshot: Match user", () => {
    const store = mockStore(testUserStore)
    Object.defineProperty(window, "matchMedia", windowMedia)

    const tree = renderer
      .create(
        <Provider store={store}>
          <MemoryRouter>
            <Teams />
          </MemoryRouter>
        </Provider>
      )
      .toJSON()

    expect(tree).toMatchSnapshot()
  })
})