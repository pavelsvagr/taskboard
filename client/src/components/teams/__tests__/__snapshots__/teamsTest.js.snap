// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Teams.jsx Snapshot: Match admin 1`] = `
<div
  className="ant-spin-nested-loading"
>
  <div
    className="ant-spin-container"
  >
    <h1>
      Teams
    </h1>
    <div
      className="ant-table-wrapper"
    >
      <div
        className="ant-spin-nested-loading"
      >
        <div
          className="ant-spin-container"
        >
          <div
            className="ant-table"
          >
            <div
              className="ant-table-container"
            >
              <div
                className="ant-table-content"
                onScroll={[Function]}
                style={Object {}}
              >
                <table
                  style={
                    Object {
                      "tableLayout": "auto",
                    }
                  }
                >
                  <colgroup />
                  <thead
                    className="ant-table-thead"
                  >
                    <tr>
                      <th
                        className="ant-table-cell ant-table-column-has-sorters"
                        colSpan={null}
                        onClick={[Function]}
                        rowSpan={null}
                        style={Object {}}
                      >
                        <div
                          className="ant-table-column-sorters"
                        >
                          <span>
                            Name
                          </span>
                          <span
                            className="ant-table-column-sorter ant-table-column-sorter-full"
                          >
                            <span
                              className="ant-table-column-sorter-inner"
                            >
                              <span
                                aria-label="caret-up"
                                className="anticon anticon-caret-up ant-table-column-sorter-up"
                                role="img"
                              >
                                <svg
                                  aria-hidden="true"
                                  data-icon="caret-up"
                                  fill="currentColor"
                                  focusable="false"
                                  height="1em"
                                  viewBox="0 0 1024 1024"
                                  width="1em"
                                >
                                  <path
                                    d="M858.9 689L530.5 308.2c-9.4-10.9-27.5-10.9-37 0L165.1 689c-12.2 14.2-1.2 35 18.5 35h656.8c19.7 0 30.7-20.8 18.5-35z"
                                  />
                                </svg>
                              </span>
                              <span
                                aria-label="caret-down"
                                className="anticon anticon-caret-down ant-table-column-sorter-down"
                                role="img"
                              >
                                <svg
                                  aria-hidden="true"
                                  data-icon="caret-down"
                                  fill="currentColor"
                                  focusable="false"
                                  height="1em"
                                  viewBox="0 0 1024 1024"
                                  width="1em"
                                >
                                  <path
                                    d="M840.4 300H183.6c-19.7 0-30.7 20.8-18.5 35l328.4 380.8c9.4 10.9 27.5 10.9 37 0L858.9 335c12.2-14.2 1.2-35-18.5-35z"
                                  />
                                </svg>
                              </span>
                            </span>
                          </span>
                        </div>
                      </th>
                      <th
                        className="ant-table-cell ant-table-column-has-sorters"
                        colSpan={null}
                        onClick={[Function]}
                        rowSpan={null}
                        style={Object {}}
                      >
                        <div
                          className="ant-table-column-sorters"
                        >
                          <span>
                            Color
                          </span>
                          <span
                            className="ant-table-column-sorter ant-table-column-sorter-full"
                          >
                            <span
                              className="ant-table-column-sorter-inner"
                            >
                              <span
                                aria-label="caret-up"
                                className="anticon anticon-caret-up ant-table-column-sorter-up"
                                role="img"
                              >
                                <svg
                                  aria-hidden="true"
                                  data-icon="caret-up"
                                  fill="currentColor"
                                  focusable="false"
                                  height="1em"
                                  viewBox="0 0 1024 1024"
                                  width="1em"
                                >
                                  <path
                                    d="M858.9 689L530.5 308.2c-9.4-10.9-27.5-10.9-37 0L165.1 689c-12.2 14.2-1.2 35 18.5 35h656.8c19.7 0 30.7-20.8 18.5-35z"
                                  />
                                </svg>
                              </span>
                              <span
                                aria-label="caret-down"
                                className="anticon anticon-caret-down ant-table-column-sorter-down"
                                role="img"
                              >
                                <svg
                                  aria-hidden="true"
                                  data-icon="caret-down"
                                  fill="currentColor"
                                  focusable="false"
                                  height="1em"
                                  viewBox="0 0 1024 1024"
                                  width="1em"
                                >
                                  <path
                                    d="M840.4 300H183.6c-19.7 0-30.7 20.8-18.5 35l328.4 380.8c9.4 10.9 27.5 10.9 37 0L858.9 335c12.2-14.2 1.2-35-18.5-35z"
                                  />
                                </svg>
                              </span>
                            </span>
                          </span>
                        </div>
                      </th>
                      <th
                        className="ant-table-cell"
                        colSpan={null}
                        rowSpan={null}
                        style={Object {}}
                      >
                        Actions
                      </th>
                    </tr>
                  </thead>
                  <tbody
                    className="ant-table-tbody"
                  >
                    <tr
                      className="ant-table-row ant-table-row-level-0"
                      data-row-key="test-smoke-team"
                      onClick={[Function]}
                      style={Object {}}
                    >
                      <td
                        className="ant-table-cell"
                        colSpan={null}
                        rowSpan={null}
                        style={Object {}}
                      >
                        test-smoke-team
                      </td>
                      <td
                        className="ant-table-cell"
                        colSpan={null}
                        rowSpan={null}
                        style={Object {}}
                      >
                        <span
                          className="ant-avatar ant-avatar-circle ant-avatar-icon color-picker--disabled color-white"
                          style={Object {}}
                        >
                          <span />
                        </span>
                      </td>
                      <td
                        className="ant-table-cell"
                        colSpan={null}
                        rowSpan={null}
                        style={Object {}}
                      >
                        <div
                          className="table__actions"
                        >
                          <button
                            className="ant-btn ant-btn-circle ant-btn-icon-only"
                            onClick={[Function]}
                            onMouseEnter={[Function]}
                            onMouseLeave={[Function]}
                            type="button"
                          >
                            <span
                              aria-label="team"
                              className="anticon anticon-team"
                              role="img"
                            >
                              <svg
                                aria-hidden="true"
                                className=""
                                data-icon="team"
                                fill="currentColor"
                                focusable="false"
                                height="1em"
                                viewBox="64 64 896 896"
                                width="1em"
                              >
                                <path
                                  d="M824.2 699.9a301.55 301.55 0 00-86.4-60.4C783.1 602.8 812 546.8 812 484c0-110.8-92.4-201.7-203.2-200-109.1 1.7-197 90.6-197 200 0 62.8 29 118.8 74.2 155.5a300.95 300.95 0 00-86.4 60.4C345 754.6 314 826.8 312 903.8a8 8 0 008 8.2h56c4.3 0 7.9-3.4 8-7.7 1.9-58 25.4-112.3 66.7-153.5A226.62 226.62 0 01612 684c60.9 0 118.2 23.7 161.3 66.8C814.5 792 838 846.3 840 904.3c.1 4.3 3.7 7.7 8 7.7h56a8 8 0 008-8.2c-2-77-33-149.2-87.8-203.9zM612 612c-34.2 0-66.4-13.3-90.5-37.5a126.86 126.86 0 01-37.5-91.8c.3-32.8 13.4-64.5 36.3-88 24-24.6 56.1-38.3 90.4-38.7 33.9-.3 66.8 12.9 91 36.6 24.8 24.3 38.4 56.8 38.4 91.4 0 34.2-13.3 66.3-37.5 90.5A127.3 127.3 0 01612 612zM361.5 510.4c-.9-8.7-1.4-17.5-1.4-26.4 0-15.9 1.5-31.4 4.3-46.5.7-3.6-1.2-7.3-4.5-8.8-13.6-6.1-26.1-14.5-36.9-25.1a127.54 127.54 0 01-38.7-95.4c.9-32.1 13.8-62.6 36.3-85.6 24.7-25.3 57.9-39.1 93.2-38.7 31.9.3 62.7 12.6 86 34.4 7.9 7.4 14.7 15.6 20.4 24.4 2 3.1 5.9 4.4 9.3 3.2 17.6-6.1 36.2-10.4 55.3-12.4 5.6-.6 8.8-6.6 6.3-11.6-32.5-64.3-98.9-108.7-175.7-109.9-110.9-1.7-203.3 89.2-203.3 199.9 0 62.8 28.9 118.8 74.2 155.5-31.8 14.7-61.1 35-86.5 60.4-54.8 54.7-85.8 126.9-87.8 204a8 8 0 008 8.2h56.1c4.3 0 7.9-3.4 8-7.7 1.9-58 25.4-112.3 66.7-153.5 29.4-29.4 65.4-49.8 104.7-59.7 3.9-1 6.5-4.7 6-8.7z"
                                />
                              </svg>
                            </span>
                          </button>
                          <button
                            className="ant-btn ant-btn-circle ant-btn-icon-only"
                            onClick={[Function]}
                            onMouseEnter={[Function]}
                            onMouseLeave={[Function]}
                            type="button"
                          >
                            <span
                              aria-label="edit"
                              className="anticon anticon-edit"
                              role="img"
                            >
                              <svg
                                aria-hidden="true"
                                className=""
                                data-icon="edit"
                                fill="currentColor"
                                focusable="false"
                                height="1em"
                                viewBox="64 64 896 896"
                                width="1em"
                              >
                                <path
                                  d="M257.7 752c2 0 4-.2 6-.5L431.9 722c2-.4 3.9-1.3 5.3-2.8l423.9-423.9a9.96 9.96 0 000-14.1L694.9 114.9c-1.9-1.9-4.4-2.9-7.1-2.9s-5.2 1-7.1 2.9L256.8 538.8c-1.5 1.5-2.4 3.3-2.8 5.3l-29.5 168.2a33.5 33.5 0 009.4 29.8c6.6 6.4 14.9 9.9 23.8 9.9zm67.4-174.4L687.8 215l73.3 73.3-362.7 362.6-88.9 15.7 15.6-89zM880 836H144c-17.7 0-32 14.3-32 32v36c0 4.4 3.6 8 8 8h784c4.4 0 8-3.6 8-8v-36c0-17.7-14.3-32-32-32z"
                                />
                              </svg>
                            </span>
                          </button>
                          <button
                            className="ant-btn ant-btn-circle ant-btn-icon-only ant-btn-dangerous"
                            onClick={[Function]}
                            onMouseEnter={[Function]}
                            onMouseLeave={[Function]}
                            type="button"
                          >
                            <span
                              aria-label="delete"
                              className="anticon anticon-delete"
                              role="img"
                            >
                              <svg
                                aria-hidden="true"
                                className=""
                                data-icon="delete"
                                fill="currentColor"
                                focusable="false"
                                height="1em"
                                viewBox="64 64 896 896"
                                width="1em"
                              >
                                <path
                                  d="M360 184h-8c4.4 0 8-3.6 8-8v8h304v-8c0 4.4 3.6 8 8 8h-8v72h72v-80c0-35.3-28.7-64-64-64H352c-35.3 0-64 28.7-64 64v80h72v-72zm504 72H160c-17.7 0-32 14.3-32 32v32c0 4.4 3.6 8 8 8h60.4l24.7 523c1.6 34.1 29.8 61 63.9 61h454c34.2 0 62.3-26.8 63.9-61l24.7-523H888c4.4 0 8-3.6 8-8v-32c0-17.7-14.3-32-32-32zM731.3 840H292.7l-24.2-512h487l-24.2 512z"
                                />
                              </svg>
                            </span>
                          </button>
                        </div>
                      </td>
                    </tr>
                  </tbody>
                </table>
              </div>
            </div>
          </div>
          <ul
            className="ant-pagination ant-table-pagination ant-table-pagination-center"
            style={Object {}}
            unselectable="unselectable"
          >
            <li
              aria-disabled={true}
              className="ant-pagination-prev ant-pagination-disabled"
              onClick={[Function]}
              onKeyPress={[Function]}
              tabIndex={null}
              title="Previous Page"
            >
              <button
                className="ant-pagination-item-link"
                disabled={true}
                tabIndex={-1}
                type="button"
              >
                <span
                  aria-label="left"
                  className="anticon anticon-left"
                  role="img"
                >
                  <svg
                    aria-hidden="true"
                    data-icon="left"
                    fill="currentColor"
                    focusable="false"
                    height="1em"
                    viewBox="64 64 896 896"
                    width="1em"
                  >
                    <path
                      d="M724 218.3V141c0-6.7-7.7-10.4-12.9-6.3L260.3 486.8a31.86 31.86 0 000 50.3l450.8 352.1c5.3 4.1 12.9.4 12.9-6.3v-77.3c0-4.9-2.3-9.6-6.1-12.6l-360-281 360-281.1c3.8-3 6.1-7.7 6.1-12.6z"
                    />
                  </svg>
                </span>
              </button>
            </li>
            <li
              className="ant-pagination-item ant-pagination-item-1 ant-pagination-item-active"
              onClick={[Function]}
              onKeyPress={[Function]}
              tabIndex="0"
              title={1}
            >
              <a
                rel="nofollow"
              >
                1
              </a>
            </li>
            <li
              aria-disabled={true}
              className="ant-pagination-next ant-pagination-disabled"
              onClick={[Function]}
              onKeyPress={[Function]}
              tabIndex={null}
              title="Next Page"
            >
              <button
                className="ant-pagination-item-link"
                disabled={true}
                tabIndex={-1}
                type="button"
              >
                <span
                  aria-label="right"
                  className="anticon anticon-right"
                  role="img"
                >
                  <svg
                    aria-hidden="true"
                    data-icon="right"
                    fill="currentColor"
                    focusable="false"
                    height="1em"
                    viewBox="64 64 896 896"
                    width="1em"
                  >
                    <path
                      d="M765.7 486.8L314.9 134.7A7.97 7.97 0 00302 141v77.3c0 4.9 2.3 9.6 6.1 12.6l360 281.1-360 281.1c-3.9 3-6.1 7.7-6.1 12.6V883c0 6.7 7.7 10.4 12.9 6.3l450.8-352.1a31.96 31.96 0 000-50.4z"
                    />
                  </svg>
                </span>
              </button>
            </li>
          </ul>
        </div>
      </div>
    </div>
    <div
      className="button-action"
      style={
        Object {
          "textAlign": "right",
        }
      }
    >
      <div
        className=""
      >
        <button
          className="ant-btn ant-btn-primary ant-btn-circle ant-btn-icon-only button"
          onClick={[Function]}
          type="button"
        >
          <span
            aria-label="plus"
            className="anticon anticon-plus"
            role="img"
          >
            <svg
              aria-hidden="true"
              className=""
              data-icon="plus"
              fill="currentColor"
              focusable="false"
              height="1em"
              viewBox="64 64 896 896"
              width="1em"
            >
              <defs>
                <style />
              </defs>
              <path
                d="M482 152h60q8 0 8 8v704q0 8-8 8h-60q-8 0-8-8V160q0-8 8-8z"
              />
              <path
                d="M176 474h672q8 0 8 8v60q0 8-8 8H176q-8 0-8-8v-60q0-8 8-8z"
              />
            </svg>
          </span>
        </button>
      </div>
    </div>
  </div>
</div>
`;

exports[`Teams.jsx Snapshot: Match user 1`] = `
<div
  className="ant-spin-nested-loading"
>
  <div
    className="ant-spin-container"
  >
    <h1>
      Teams
    </h1>
    <div
      className="ant-table-wrapper"
    >
      <div
        className="ant-spin-nested-loading"
      >
        <div
          className="ant-spin-container"
        >
          <div
            className="ant-table"
          >
            <div
              className="ant-table-container"
            >
              <div
                className="ant-table-content"
                onScroll={[Function]}
                style={Object {}}
              >
                <table
                  style={
                    Object {
                      "tableLayout": "auto",
                    }
                  }
                >
                  <colgroup />
                  <thead
                    className="ant-table-thead"
                  >
                    <tr>
                      <th
                        className="ant-table-cell ant-table-column-has-sorters"
                        colSpan={null}
                        onClick={[Function]}
                        rowSpan={null}
                        style={Object {}}
                      >
                        <div
                          className="ant-table-column-sorters"
                        >
                          <span>
                            Name
                          </span>
                          <span
                            className="ant-table-column-sorter ant-table-column-sorter-full"
                          >
                            <span
                              className="ant-table-column-sorter-inner"
                            >
                              <span
                                aria-label="caret-up"
                                className="anticon anticon-caret-up ant-table-column-sorter-up"
                                role="img"
                              >
                                <svg
                                  aria-hidden="true"
                                  data-icon="caret-up"
                                  fill="currentColor"
                                  focusable="false"
                                  height="1em"
                                  viewBox="0 0 1024 1024"
                                  width="1em"
                                >
                                  <path
                                    d="M858.9 689L530.5 308.2c-9.4-10.9-27.5-10.9-37 0L165.1 689c-12.2 14.2-1.2 35 18.5 35h656.8c19.7 0 30.7-20.8 18.5-35z"
                                  />
                                </svg>
                              </span>
                              <span
                                aria-label="caret-down"
                                className="anticon anticon-caret-down ant-table-column-sorter-down"
                                role="img"
                              >
                                <svg
                                  aria-hidden="true"
                                  data-icon="caret-down"
                                  fill="currentColor"
                                  focusable="false"
                                  height="1em"
                                  viewBox="0 0 1024 1024"
                                  width="1em"
                                >
                                  <path
                                    d="M840.4 300H183.6c-19.7 0-30.7 20.8-18.5 35l328.4 380.8c9.4 10.9 27.5 10.9 37 0L858.9 335c12.2-14.2 1.2-35-18.5-35z"
                                  />
                                </svg>
                              </span>
                            </span>
                          </span>
                        </div>
                      </th>
                      <th
                        className="ant-table-cell ant-table-column-has-sorters"
                        colSpan={null}
                        onClick={[Function]}
                        rowSpan={null}
                        style={Object {}}
                      >
                        <div
                          className="ant-table-column-sorters"
                        >
                          <span>
                            Color
                          </span>
                          <span
                            className="ant-table-column-sorter ant-table-column-sorter-full"
                          >
                            <span
                              className="ant-table-column-sorter-inner"
                            >
                              <span
                                aria-label="caret-up"
                                className="anticon anticon-caret-up ant-table-column-sorter-up"
                                role="img"
                              >
                                <svg
                                  aria-hidden="true"
                                  data-icon="caret-up"
                                  fill="currentColor"
                                  focusable="false"
                                  height="1em"
                                  viewBox="0 0 1024 1024"
                                  width="1em"
                                >
                                  <path
                                    d="M858.9 689L530.5 308.2c-9.4-10.9-27.5-10.9-37 0L165.1 689c-12.2 14.2-1.2 35 18.5 35h656.8c19.7 0 30.7-20.8 18.5-35z"
                                  />
                                </svg>
                              </span>
                              <span
                                aria-label="caret-down"
                                className="anticon anticon-caret-down ant-table-column-sorter-down"
                                role="img"
                              >
                                <svg
                                  aria-hidden="true"
                                  data-icon="caret-down"
                                  fill="currentColor"
                                  focusable="false"
                                  height="1em"
                                  viewBox="0 0 1024 1024"
                                  width="1em"
                                >
                                  <path
                                    d="M840.4 300H183.6c-19.7 0-30.7 20.8-18.5 35l328.4 380.8c9.4 10.9 27.5 10.9 37 0L858.9 335c12.2-14.2 1.2-35-18.5-35z"
                                  />
                                </svg>
                              </span>
                            </span>
                          </span>
                        </div>
                      </th>
                      <th
                        className="ant-table-cell"
                        colSpan={null}
                        rowSpan={null}
                        style={Object {}}
                      >
                        Actions
                      </th>
                    </tr>
                  </thead>
                  <tbody
                    className="ant-table-tbody"
                  >
                    <tr
                      className="ant-table-row ant-table-row-level-0"
                      data-row-key="test-smoke-team"
                      onClick={[Function]}
                      style={Object {}}
                    >
                      <td
                        className="ant-table-cell"
                        colSpan={null}
                        rowSpan={null}
                        style={Object {}}
                      >
                        test-smoke-team
                      </td>
                      <td
                        className="ant-table-cell"
                        colSpan={null}
                        rowSpan={null}
                        style={Object {}}
                      >
                        <span
                          className="ant-avatar ant-avatar-circle ant-avatar-icon color-picker--disabled color-white"
                          style={Object {}}
                        >
                          <span />
                        </span>
                      </td>
                      <td
                        className="ant-table-cell"
                        colSpan={null}
                        rowSpan={null}
                        style={Object {}}
                      >
                        <div
                          className="table__actions"
                        >
                          <button
                            className="ant-btn ant-btn-circle ant-btn-icon-only"
                            onClick={[Function]}
                            onMouseEnter={[Function]}
                            onMouseLeave={[Function]}
                            type="button"
                          >
                            <span
                              aria-label="team"
                              className="anticon anticon-team"
                              role="img"
                            >
                              <svg
                                aria-hidden="true"
                                className=""
                                data-icon="team"
                                fill="currentColor"
                                focusable="false"
                                height="1em"
                                viewBox="64 64 896 896"
                                width="1em"
                              >
                                <path
                                  d="M824.2 699.9a301.55 301.55 0 00-86.4-60.4C783.1 602.8 812 546.8 812 484c0-110.8-92.4-201.7-203.2-200-109.1 1.7-197 90.6-197 200 0 62.8 29 118.8 74.2 155.5a300.95 300.95 0 00-86.4 60.4C345 754.6 314 826.8 312 903.8a8 8 0 008 8.2h56c4.3 0 7.9-3.4 8-7.7 1.9-58 25.4-112.3 66.7-153.5A226.62 226.62 0 01612 684c60.9 0 118.2 23.7 161.3 66.8C814.5 792 838 846.3 840 904.3c.1 4.3 3.7 7.7 8 7.7h56a8 8 0 008-8.2c-2-77-33-149.2-87.8-203.9zM612 612c-34.2 0-66.4-13.3-90.5-37.5a126.86 126.86 0 01-37.5-91.8c.3-32.8 13.4-64.5 36.3-88 24-24.6 56.1-38.3 90.4-38.7 33.9-.3 66.8 12.9 91 36.6 24.8 24.3 38.4 56.8 38.4 91.4 0 34.2-13.3 66.3-37.5 90.5A127.3 127.3 0 01612 612zM361.5 510.4c-.9-8.7-1.4-17.5-1.4-26.4 0-15.9 1.5-31.4 4.3-46.5.7-3.6-1.2-7.3-4.5-8.8-13.6-6.1-26.1-14.5-36.9-25.1a127.54 127.54 0 01-38.7-95.4c.9-32.1 13.8-62.6 36.3-85.6 24.7-25.3 57.9-39.1 93.2-38.7 31.9.3 62.7 12.6 86 34.4 7.9 7.4 14.7 15.6 20.4 24.4 2 3.1 5.9 4.4 9.3 3.2 17.6-6.1 36.2-10.4 55.3-12.4 5.6-.6 8.8-6.6 6.3-11.6-32.5-64.3-98.9-108.7-175.7-109.9-110.9-1.7-203.3 89.2-203.3 199.9 0 62.8 28.9 118.8 74.2 155.5-31.8 14.7-61.1 35-86.5 60.4-54.8 54.7-85.8 126.9-87.8 204a8 8 0 008 8.2h56.1c4.3 0 7.9-3.4 8-7.7 1.9-58 25.4-112.3 66.7-153.5 29.4-29.4 65.4-49.8 104.7-59.7 3.9-1 6.5-4.7 6-8.7z"
                                />
                              </svg>
                            </span>
                          </button>
                        </div>
                      </td>
                    </tr>
                    <tr
                      className="ant-table-row ant-table-row-level-0"
                      data-row-key="test-smoke-team2"
                      onClick={[Function]}
                      style={Object {}}
                    >
                      <td
                        className="ant-table-cell"
                        colSpan={null}
                        rowSpan={null}
                        style={Object {}}
                      >
                        test-smoke-team2
                      </td>
                      <td
                        className="ant-table-cell"
                        colSpan={null}
                        rowSpan={null}
                        style={Object {}}
                      >
                        <span
                          className="ant-avatar ant-avatar-circle ant-avatar-icon color-picker--disabled color-blue"
                          style={Object {}}
                        >
                          <span />
                        </span>
                      </td>
                      <td
                        className="ant-table-cell"
                        colSpan={null}
                        rowSpan={null}
                        style={Object {}}
                      >
                        <div
                          className="table__actions"
                        >
                          <button
                            className="ant-btn ant-btn-circle ant-btn-icon-only"
                            onClick={[Function]}
                            onMouseEnter={[Function]}
                            onMouseLeave={[Function]}
                            type="button"
                          >
                            <span
                              aria-label="team"
                              className="anticon anticon-team"
                              role="img"
                            >
                              <svg
                                aria-hidden="true"
                                className=""
                                data-icon="team"
                                fill="currentColor"
                                focusable="false"
                                height="1em"
                                viewBox="64 64 896 896"
                                width="1em"
                              >
                                <path
                                  d="M824.2 699.9a301.55 301.55 0 00-86.4-60.4C783.1 602.8 812 546.8 812 484c0-110.8-92.4-201.7-203.2-200-109.1 1.7-197 90.6-197 200 0 62.8 29 118.8 74.2 155.5a300.95 300.95 0 00-86.4 60.4C345 754.6 314 826.8 312 903.8a8 8 0 008 8.2h56c4.3 0 7.9-3.4 8-7.7 1.9-58 25.4-112.3 66.7-153.5A226.62 226.62 0 01612 684c60.9 0 118.2 23.7 161.3 66.8C814.5 792 838 846.3 840 904.3c.1 4.3 3.7 7.7 8 7.7h56a8 8 0 008-8.2c-2-77-33-149.2-87.8-203.9zM612 612c-34.2 0-66.4-13.3-90.5-37.5a126.86 126.86 0 01-37.5-91.8c.3-32.8 13.4-64.5 36.3-88 24-24.6 56.1-38.3 90.4-38.7 33.9-.3 66.8 12.9 91 36.6 24.8 24.3 38.4 56.8 38.4 91.4 0 34.2-13.3 66.3-37.5 90.5A127.3 127.3 0 01612 612zM361.5 510.4c-.9-8.7-1.4-17.5-1.4-26.4 0-15.9 1.5-31.4 4.3-46.5.7-3.6-1.2-7.3-4.5-8.8-13.6-6.1-26.1-14.5-36.9-25.1a127.54 127.54 0 01-38.7-95.4c.9-32.1 13.8-62.6 36.3-85.6 24.7-25.3 57.9-39.1 93.2-38.7 31.9.3 62.7 12.6 86 34.4 7.9 7.4 14.7 15.6 20.4 24.4 2 3.1 5.9 4.4 9.3 3.2 17.6-6.1 36.2-10.4 55.3-12.4 5.6-.6 8.8-6.6 6.3-11.6-32.5-64.3-98.9-108.7-175.7-109.9-110.9-1.7-203.3 89.2-203.3 199.9 0 62.8 28.9 118.8 74.2 155.5-31.8 14.7-61.1 35-86.5 60.4-54.8 54.7-85.8 126.9-87.8 204a8 8 0 008 8.2h56.1c4.3 0 7.9-3.4 8-7.7 1.9-58 25.4-112.3 66.7-153.5 29.4-29.4 65.4-49.8 104.7-59.7 3.9-1 6.5-4.7 6-8.7z"
                                />
                              </svg>
                            </span>
                          </button>
                        </div>
                      </td>
                    </tr>
                  </tbody>
                </table>
              </div>
            </div>
          </div>
          <ul
            className="ant-pagination ant-table-pagination ant-table-pagination-center"
            style={Object {}}
            unselectable="unselectable"
          >
            <li
              aria-disabled={true}
              className="ant-pagination-prev ant-pagination-disabled"
              onClick={[Function]}
              onKeyPress={[Function]}
              tabIndex={null}
              title="Previous Page"
            >
              <button
                className="ant-pagination-item-link"
                disabled={true}
                tabIndex={-1}
                type="button"
              >
                <span
                  aria-label="left"
                  className="anticon anticon-left"
                  role="img"
                >
                  <svg
                    aria-hidden="true"
                    data-icon="left"
                    fill="currentColor"
                    focusable="false"
                    height="1em"
                    viewBox="64 64 896 896"
                    width="1em"
                  >
                    <path
                      d="M724 218.3V141c0-6.7-7.7-10.4-12.9-6.3L260.3 486.8a31.86 31.86 0 000 50.3l450.8 352.1c5.3 4.1 12.9.4 12.9-6.3v-77.3c0-4.9-2.3-9.6-6.1-12.6l-360-281 360-281.1c3.8-3 6.1-7.7 6.1-12.6z"
                    />
                  </svg>
                </span>
              </button>
            </li>
            <li
              className="ant-pagination-item ant-pagination-item-1 ant-pagination-item-active"
              onClick={[Function]}
              onKeyPress={[Function]}
              tabIndex="0"
              title={1}
            >
              <a
                rel="nofollow"
              >
                1
              </a>
            </li>
            <li
              aria-disabled={true}
              className="ant-pagination-next ant-pagination-disabled"
              onClick={[Function]}
              onKeyPress={[Function]}
              tabIndex={null}
              title="Next Page"
            >
              <button
                className="ant-pagination-item-link"
                disabled={true}
                tabIndex={-1}
                type="button"
              >
                <span
                  aria-label="right"
                  className="anticon anticon-right"
                  role="img"
                >
                  <svg
                    aria-hidden="true"
                    data-icon="right"
                    fill="currentColor"
                    focusable="false"
                    height="1em"
                    viewBox="64 64 896 896"
                    width="1em"
                  >
                    <path
                      d="M765.7 486.8L314.9 134.7A7.97 7.97 0 00302 141v77.3c0 4.9 2.3 9.6 6.1 12.6l360 281.1-360 281.1c-3.9 3-6.1 7.7-6.1 12.6V883c0 6.7 7.7 10.4 12.9 6.3l450.8-352.1a31.96 31.96 0 000-50.4z"
                    />
                  </svg>
                </span>
              </button>
            </li>
          </ul>
        </div>
      </div>
    </div>
  </div>
</div>
`;
